README.md

```diff
@@ -0,0 +1,34 @@
+# Setting up Developer Environment
+
+## Creating Developer Environment
+For Mac(Intel)
+
+### Steps
+
+1. Start Docker daemon, install anaconda3
+
+2. Create an environment for Feast, selecting python 3.9. Activate the environment: 
+```
+conda create --name feast python=3.9
+conda activate feast
+```
+
+3. Install dependencies:
+```
+pip install pip-tools
+brew install mysql@8.0 (latest - mysql@8.3 version fails the dev env)
+brew install xz protobuf openssl zlib
+pip install cryptography -U
+conda install protobuf
+conda install pymssql
+pip install -e ".[dev]"
```

Comment:
I've hit an error at this step, do you recall if you hit this as well? Could just be local to me, but thought I'd check
<img width="1478" alt="image" src="https://github.com/RHEcosystemAppEng/feast-workshop-team-share/assets/1012188/0d83120c-0114-42f2-96aa-6bad1de90135">

Reply:
no worries, I was able to work thru it and validate `make` steps, just wanted to be sure it was just my enviro. This LGTM, thanks!